.github/workflows/ci-main.yml

```diff
@@ -123,7 +123,7 @@ jobs:
         uses: docker/build-push-action@v5
         with:
           context: .
-          platforms: linux/amd64,linux/arm64
+          platforms: linux/amd64,linux/arm64,linux/arm/v7
```

Comment:
I know this is outside the scope of your interest but will this also support most 32-bit Raspberry Pis?  I believe that only the early ones were v6: https://raspberrypi.stackexchange.com/a/139499